src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -144,6 +151,15 @@ function IOURequestStepParticipants({
         IOUUtils.navigateToStartMoneyRequestStep(iouRequestType, iouType, transactionID, reportID, action);
     }, [iouRequestType, iouType, transactionID, reportID, action]);
 
+    const trackExpense = () => {
+        // If coming from the combined submit/track flow and the user proceeds to just track the expense,
+        // we will use the track IOU type in the confirmation flow.
+        const selfDMReport = ReportUtils.findSelfDMReport();
```

Comment:
See https://github.com/Expensify/App/blob/main/src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx#L375-L381

Reply:
We need to cover the case when a user doesnt have a selfDM

Reply:
As discussed in Slack we agreed to hide the "Just Track It" button if the user doesn't have a Self DM -- we have the same behavior for the Track Expense button in the Global Create menu.